src/utils/internal/getCallFrame.ts

```diff
@@ -2,25 +2,21 @@
  * Return the stack trace frame of a function's invocation.
  */
 export function getCallFrame() {
-  try {
-    const inspectionError = new Error()
-    inspectionError.name = 'Inspection Error'
-    throw inspectionError
-  } catch (error) {
-    const frames: string[] = error.stack.split('\n')
+  // Error.stack is available even in IE 10, no need for nullish coalesce
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack#browser_compatibility
+  const frames: string[] = new Error().stack!.split('\n')
```

Comment:
Do we need `!` here because `Error.stack` is typed as optional in TypeScript?

Reply:
yes. TS definitions for web apis are quite out of date TBH

P.S.: I don't like that `Resolve conversation` button does deletes my draft reply

Reply:
looks like you guys disabled `!` in eslint => I am returning back to my original `as string` for now

<img width="602" alt="Screenshot 2021-01-29 at 19 46 06" src="https://user-images.githubusercontent.com/1087670/106314915-b7860080-626a-11eb-869a-6bb7e9d4d430.png">